pages/flow/index.js

```diff
@@ -1,35 +1,156 @@
-              />
-            </li>
-          );
-        })}
+        {flowAsanas.map((asana, index) => (
+          <li key={index}>
+            <AsanaCard
+              name={asana.english_name}
+              img={asana.img_url}
+              id={asana.id}
+            />
+          </li>
+        ))}
       </StyledList>
-    </>
+
+      {!open && (
+        <StyledButton onClick={() => setOpen(true)}>
+          Create your Flow
+        </StyledButton>
+      )}
+
+      {open && (
+        <AddAsanaSection>
+          <SectionHeader>
+            <CloseButton onClick={() => setOpen(false)}>X</CloseButton>
+            <StyledCounter>{countAsanas}</StyledCounter>
+          </SectionHeader>
+          <StyledList>
+            {asanas.map((asana) => (
+              <StyledListItem key={asana.id}>
+                <h3>{asana.english_name}</h3>
+                <StyledAddButton
+                  onClick={() => {
+                    setFlowAsanaIds([...flowAsanaIds, asana.id]);
+                    sumUpAsanas();
+                  }}
+                >
+                  <a>+</a>
+                </StyledAddButton>
+              </StyledListItem>
+            ))}
+          </StyledList>
+        </AddAsanaSection>
+      )}
+    </StyledContainer>
   );
 }
 
 const StyledList = styled.ul`
   list-style: none;
 `;
+
+const StyledListItem = styled.li`
+  display: flex;
+  gap: 2rem;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const StyledAddButton = styled.div`
+  all: unset;
+  cursor: pointer;
+  background-color: #5d6bea;
+  color: #f5f5f5;
+  padding: 0;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  text-align: center;
+  &:active {
+    background-color: orange;
+  }
+`;
+
+const AddAsanaSection = styled.section`
+  position: absolute;
+  top: 4rem;
+  right: 0;
+  height: 80vh;
+  background-color: #ffffff;
+  border-radius: 22px;
+  padding: 1.5rem;
+  z-index: 5;
+  box-shadow: 0px 0px 4px rgba(93, 107, 234, 0.42);
+  overflow-y: scroll;
+`;
+
+const StyledH2 = styled.h2`
+  text-align: center;
+`;
+
+const StyledButton = styled.button`
+  width: 16rem;
+  margin: 2rem auto;
+  display: block;
+  border: none;
+  background-color: #ff8667;
+  color: #f5f5f5;
+  padding: 1rem 3rem;
+  border-radius: 22px;
+  font-size: 1.1rem;
+  cursor: pointer;
+`;
+
+const StyledContainer = styled.section`
+  position: relative;
+`;
+
+const CloseButton = styled.div`
+  background-color: #5d6bea;
+  color: #f5f5f5;
+  padding: 0;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  text-align: center;
+  cursor: pointer; ;
+`;
+
+const StyledCounter = styled.div`
+  background-color: #ff8667;
+  color: #f5f5f5;
+  padding: 0;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 22px;
+  text-align: center;
+`;
+
+const SectionHeader = styled.section`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  padding-bottom: 1rem;
+  border-bottom: 1px solid grey;
```

Comment:
never use colors like `red`. Use Hex-Colors or RGB or HSL.
Colors like `red` can be interpreted differently by different browsers

Pls look inside all styled Components and correct this